.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - "jruby"
+script:
+  - bundle exec rake all
```

Comment:
I think with a  one line script you can just use:

```yaml
script: "bundle exec rake all"
```

...again, you might not need the quotes there